qml/DynamicDisplayStatisticsDialog.qml

```diff
@@ -51,18 +55,38 @@ Dialog {
         regenerateSeriesLabel()
     }
 
+    onAboutToShow: {
```

Comment:
I would suggest in onCompleted to call a function `updateAllEntities` that actually _update all Entity kinds_ and do not manage new refreshes while the Dialog is open (this is very reasonable, not to refresh an already opened dialog every time).
The way it is done now will call too much times to Backend `get_entities`, a non cheap backend function

Reply:
Yes but the dialog will be opened only a few times. And do not refresh the entities everytime the dialog is opened may cause the bug that we saw yesterday.

Reply:
I'll add here the answers to your suggestions:
Suggestions:
- _Add `Please choose a DataKind` in `Dynamic/HistoricDataKindDialog`_
  - Addressed in 3356e39
- _Reduce calls to refresh entities while in a Dialog (right now it does not refresh whitout changin the EntityKind, so why not refresh at all while open)_
  - Doing that may cause some bugs.
- if possible: allow clicking in an entity (left sidebar) all along its horiontal bar (at least if it is clicked, as the horiontal bar goes till the end of the Panel but only clicking in the name is allowed)
  - This was done in 1a550a6

Reply:
> Add Please choose a DataKind in Dynamic/HistoricDataKindDialog 
It does not check if the DataKind has been selected and so it does not show an error